snap/build-scripts/docker-build

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
-docker build -t docker-build .
+set -eux
+
+docker build -f "./Dockerfile.$(uname -m)" -t docker-build .
```

Comment:
So every time we build we're going to update the build tool containers? We don't want this as a different step so we can update them, but not force it on every build?

Reply:
im down for not refreshing new containers, but i suck at docker so feel free to tack on to this